src/govuk/components/notification-banner/template.njk

```diff
@@ -0,0 +1,40 @@
+{% if params.role %}
+  {% set role = params.role %}
+{% elif params.type == "success" or params.type == "error" %}
+  {% set role = "alert" %}
+{% else %}
+  {% set role = "region" %}
+{% endif %}
+
+{% if params.tabindex %}
+  {% set tabindex = params.tabindex %}
+{% elif params.type == "success" or params.type == "error" %}
+  {% set tabindex = "-1" %}
+{% endif %}
+
+{% set titleHeadingLevel = params.titleHeadingLevel if params.titleHeadingLevel else 2 %}
+
+<div class="govuk-notification-banner
```

Comment:
It'd be good to look at the whitespace control in this template, as it's including extra whitespace and line breaks in a few places:

```html
<div class="govuk-notification-banner"
  role="region"
 aria-labelledby="govuk-notification-banner-title" >
  <div class="govuk-notification-banner__header">
    <h2 class="govuk-notification-banner__title"  id="govuk-notification-banner-title"     >Important</h2>
  </div>
  <div class="govuk-notification-banner__content">
    <h3 class="govuk-heading-m">This publication was withdrawn on 7 March 2014</h3><p>Archived and replaced by the <a href="#">new planning guidance</a> launched 6 March 2014 on an external website</p>

  </div>
</div>
```

Reply:
@36degrees This should be resolved now.

Reply:
Looking really good, although I think there's an errant `-` which is causing the space before `aria-labelledby` to get stripped:

```html
<div class="govuk-notification-banner " role="region"aria-labelledby="govuk-notification-banner-title">
  [..]
</div>
```

I think:
```
{%- if (role == "region") -%} aria-labelledby="{{ params.titleId | default('govuk-notification-banner-title') }}" {%- endif -%}
```

wants to be:

```
{%- if (role == "region") %} aria-labelledby="{{ params.titleId | default('govuk-notification-banner-title') }}" {%- endif -%}
```

Reply:
Done!